apps/browser-extension-wallet/src/hooks/useWalletManager.ts

```diff
@@ -528,6 +574,78 @@ export const useWalletManager = (): UseWalletManager => {
     [getCurrentChainId]
   );
 
+  const createSharedWallet = async ({
+    mnemonic,
+    name,
+    password,
+    chainId = getCurrentChainId(),
+    publicKeys,
```

Comment:
This name would imply that you don't have to include your own key
```suggestion
    otherCoSignerXPubKeys,
```

Reply:
This key also contains the key of the ownSigner passed [here](https://github.com/input-output-hk/lace/blob/aaf64f5c7cefe93359ba01e3cbb4a8d59267959b/apps/browser-extension-wallet/src/views/browser-view/features/shared-wallet/SharedWallet.tsx#L55-L56).